src/core/dash_test.cc

```diff
@@ -882,12 +882,11 @@ struct BlankPolicy : public BasicDashPolicy {
 // into a new segment, not every item finds a place.
 TEST_F(DashTest, SplitBug) {
   DashTable<uint64_t, uint64_t, BlankPolicy> table;
+  string path = base::ProgramRunfile("testdata/ids.txt.zst");
```

Comment:
unrelated to the PR - replaced test file with the compressed file to save disk space